modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -43,7 +42,7 @@ class GrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   {
     FILL,
     USE,
-    BYPASS
+    BYPASS,
```

Comment:
No, I had another option there during development but removed it. The comma can go. On the other hand I'm glad the compiler doesn't warn or error. Sometimes it's easy not to have a special case (no comma on the last item).